build_tools/build_aietools_distro/copy.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -euo pipefail
```

Comment:
Maybe explain what this file is for as it's not used? Or can we put this file in CI as well to avoid it going stale?